management/commands/import_plos2EA.py

```diff
@@ -0,0 +1,495 @@
+from django.conf import settings as django_settings
+from django.core.management.base import BaseCommand
+import requests
+import datetime
+import json
+import urllib.request
+import os
+import django.utils.timezone
+from submission import models as submission_models
+from repository import models as repository_models
+from core import models as core_models
+from django.template.defaultfilters import slugify
+from uuid import uuid4
+from django.core.mail import send_mail
+
+import xml.etree.ElementTree as ET
+from ftplib import FTP_TLS
+from datetime import datetime
+import traceback
+import sys
+
+PRESS_ID = 1

```

Comment:
It would be nice to make all of these config options environment variables, so they can be changed to suit. These current options could be the default, so the script would continue as-is, but then could work on stg and prd with new ENV overrides.

Reply:
The PRESS_ID and other IDs are queried from DB and used by the script. Keeping in the same file to keep all the content of the script in one place.